### ISSUE TITLE
After upload files or delete not refresh

### ISSUE THREAD

**Issue by wlorenzetti** [jsooter/RichFilemanager-Python3Flask#17]
Hi, 
after uploading action or delete action view is not refreshed and I've to refresh html page by F5 for to get news in tree and main window files view
Thanks

**Comment by git-bone** [comment]
Yep, I'm having the same issue, after upload file. Instead of f5, you can also switch between grid view and normal view, that does the trick for me. I haven't been able to find out what is causing this yet.
